vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp

```diff
@@ -29,8 +30,9 @@ $( document ).ready(function() {
 	
 	$("#addSpaceLinkButton").click(function(e) {
 		$("#bgImage").on("click", function(e){
-		    e.preventDefault();
-		    var icon = $('<span data-feather="navigation-2" class="flex"></span>');
+			e.preventDefault();			
+		    $("#arrow").remove();
```

Comment:
what if there is no element with that id? does it just ignore it or does it throw an error?

Reply:
No, it does not throw an error it is just ignoring that case.